component/src/types/options.ts

```diff
@@ -56,6 +56,8 @@ export interface AzureOptions {
   region: string;
   // https://docs.microsoft.com/azure/cognitive-services/speech-service/supported-languages
   language?: string;
+  // max 4 languages can be added (azure limitation)
```

Comment:
Can you point me towards the documentation that states that max is 4? Thankyou!

Reply:
if we try to use 5 languages, an error will appear:
`Language identification only supports 4 languages in DetectAudioAtStart mode websocket error code: 1007`
Documentation:
https://learn.microsoft.com/en-us/azure/ai-services/speech-service/language-identification?tabs=once&pivots=programming-language-javascript

> You can include up to four languages for [at-start LID](https://learn.microsoft.com/en-us/azure/ai-services/speech-service/language-identification?tabs=once&pivots=programming-language-javascript#at-start-and-continuous-language-identification) or up to 10 languages for [continuous LID](https://learn.microsoft.com/en-us/azure/ai-services/speech-service/language-identification?tabs=once&pivots=programming-language-javascript#at-start-and-continuous-language-identification).

Reply:
I updated the code now we can use up to 10 if in `Continuous` detection mode